scio-hdfs/src/main/java/com/google/cloud/dataflow/contrib/hadoop/HadoopFileSink.java

```diff
@@ -46,7 +46,6 @@
  * @param <V> The type of values to be written to the sink.
  */
 public class HadoopFileSink<K, V> extends Sink<KV<K, V>> {
-
```

Comment:
There're no other logic change in this file other than the 3 blank line deletions. I'd prefer to not let them in.